### PR TITLE
Added ingredients

### PR DESCRIPTION
**Pull request by MithulaNaik**
Added some vanilla extract

### CODE REVIEW

**Comment by dminio** [comment]
Yep - that was definitely missing. Good call!

**Comment by AndreanneTrudeau** [comment]
Yes! Good call! :)

File: ChocolateChipCookieRecipe.md [hunk 1]
```diff
@@ -8,10 +8,11 @@
 - 1/2 tsp baking powder
 - 3 TBSP water
 - 1 cup + 2 TBSP all-purpose flour
+- 2 tsp Vanilla extract
```
Comment (FourteenZ) [inline ChocolateChipCookieRecipe.md:11]: may be too much vanilla

Comment (AndreanneTrudeau) [inline ChocolateChipCookieRecipe.md:11]: sounds like the right amount of vanilla extract. great!

**Review by dminio — APPROVED** [review]
I like it - let's go ahead

**Review by FourteenZ — COMMENTED** [review]
good.

**Review by AndreanneTrudeau — APPROVED** [review]

**Pull request closed without merging**